.tekton/update-dependencies.yaml

```diff
@@ -116,35 +114,17 @@ spec:
                       name: github
                       key: token
               workingDir: $(workspaces.source.path)
-              script: |
-                #!/usr/bin/env bash
-                set -o errexit
-                set -o pipefail
-                set -x
-
-                if git diff --quiet "$(params.target_branch)"; then
-                  # No diff
-                  printf "yes" > "$(results.skip-open-pr.path)"
-                  exit
-                fi
-
-                IS_PR_CREATED=$(
-                  curl \
-                    --data-urlencode "head=$(params.repo_owner):$(params.target_branch)" \
-                    --fail \
-                    --get \
-                    --header "Accept: application/vnd.github+json" \
-                    --header "Authorization: Bearer $GITHUB_TOKEN"\
-                    --silent \
-                    "https://api.github.com/repos/$(params.repo_owner)/$(params.repo_name)/pulls" |
-                    jq ". | length"
-                )
-                if [ "$IS_PR_CREATED" = "0" ]; then
-                  printf "no"
-                else
-                  # PR already opened
-                  printf "yes"
-                fi > "$(results.skip-open-pr.path)"
+              command:
+                - $(workspaces.source.path)/developer/images/dependencies-update/hack/bin/is-pr-skipped.sh
```

Comment:
same as above

Reply:
`is-pr-skipped.sh` -> `is_pr_skipped.sh` ?

Reply:
Fixed, and file renamed to use '-' instead of '_'.